index.html

```diff
@@ -0,0 +1,104 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- CSS only -->
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+<link rel="stylesheet" href="assets/styles/style.css">
+<title>Nerd's Library</title>
+</head>
+<body>
+  <div class="form-container d-none">
+
+        <div class="mx-auto py-5 bg-custom rounded  col-md-6">
+            
+            <form class="book_input">
+                <h3 class="text-center font-weight-bolder">Add a new book to your library</h3>
+                <div class="form-row">
+                  <div class="form-group col-md-6">
+                    <label for="book_title">Book title</label>
+                    <input type="text" class="form-control" name="book_title" id="book_title">
+                  </div>
+                  <div class="form-group col-md-6">
+                    <label for="author_name">Author Name:</label>
+                    <input type="text" class="form-control" name="author_name" id="author_name">
+                  </div>
+                </div>
+                <div class="form-group">
+                    <label for="book_pages">Number of Pages</label>
+                    <input type="text" class="form-control" name="book_pages" id="book_pages">
```

Comment:
Also, the number of pages column is missing validation.
```suggestion
                    <input type="number" class="form-control" name="book_pages" id="book_pages" required>
```